### ISSUE TITLE
Upstream patched changes

### ISSUE THREAD

**Issue by fhoering** [privacysandbox/bidding-auction-local-testing-app#3]
Can you upstream the changes that have been made here https://raw.githubusercontent.com/privacysandbox/bidding-auction-local-console/refs/heads/main/resources/stack-1.patch to https://github.com/privacysandbox/bidding-auction-servers ?

It would be easier to directly use everything from the initial repo bidding-auction-servers as those changes might also break with further releases.

**Comment by kevinkiklee** [comment]
Hi Fabian, apologies for totally missing this issue until now.  

4.4 has been released, and the patch file isn't needed anymore (the changes that were made in that patch file has been incorporated into 4.4).  

~~However, the README does not reflect 4.4.  I'll update it by the EOD Mon Dec 2nd.~~ 

Edit: Wait, I actually did make the 4.4 changes before I went on a vacation.  This is what my brain is like on a Monday morning after a vacation.
